futures-util/Cargo.toml

```diff
@@ -15,8 +15,9 @@ Common utilities and extension traits for the futures-rs library.
 name = "futures_util"
 
 [features]
-std = ["alloc", "futures-core-preview/std", "futures-io-preview/std", "futures-sink-preview/std", "futures-select-macro-preview/std", "rand", "rand_core", "slab"]
+std = ["alloc", "futures-core-preview/std", "futures-io-preview/std", "futures-sink-preview/std", "futures-select-macro-preview/std", "slab"]
 default = ["std"]
+async-await = ["rand", "rand_core"]
```

Comment:
This feature also depends on `std`, right?

Reply:
Yes, it should (for now).